PortalFetch/crawler_download.py

```diff
@@ -244,8 +278,10 @@ def main():
         fillAdvanceSearch(driver)
         # Save searched courses
         html = saveResult(driver)
-        filename = parser.get('config', 'filename')
-        quarter = parser.get('config', 'quarter')
+        # get Quarter and fileName based On quarter_value in crawler.config
```

Comment:
Should be changed to `get quarter and filename based on quarter_value in crawler.config`.